packages/api-tests/test/window.spec.js

```diff
@@ -0,0 +1,92 @@
+const assert = require('assert');
+
+module.exports = (setup, mocha) => {
+  const {
+    beforeEach,
+    afterEach,
+    describe,
+    it
+  } = mocha;
+
+  let app;
+
+  describe('Window API', function(done) {
+    const timeout = 60000;
+    this.timeout(timeout);
+
+    beforeEach(() => {
+      app = setup(timeout);
+
+      return app.start();
+    });
+
+    afterEach(function() {
+      if (app && app.isRunning()) {
+        return app.stop();
+      }
+    });
+
+    const executeAsyncJavascript = (client, script) => {
+      return client.executeAsync(script);
+    };
+
+    it('Check ssf.Window is available globally', function() {
+      const script = `
+        var callback = arguments[arguments.length - 1];
+        if (ssf.Window !== undefined) {
+          callback();
+        }
+      `;
+      return executeAsyncJavascript(app.client, script);
+    });
+
+    it('Check window constructor opens a new window', function() {
+      const script = `
+        ssf.app.ready().then(() => {
+          var callback = arguments[arguments.length - 1];
+          new ssf.Window({url: 'about:blank', name: 'test', show: true, child: true});
+          setTimeout(() => callback(), 500);
+        });
+      `;
+      return executeAsyncJavascript(app.client, script).then((result) => {
+        return app.client.getWindowCount().then((count) => {
+          assert.equal(count, 2);
+        });
+      });
+    });
+
+    it('Check new window has correct x position', function() {
```

Comment:
It's worth a try. I think it's worth working on this test case to see what you could move into code that could be common across multiple tests. Why not add a case that verifies the y position as well?

(although in practice I'd verify both in one test, but at least it helps highlight the problem!)